example/example.py

```
@@ -0,0 +1,60 @@
+
+    n = 0
+    try:
+        while True:
+            my_status = f"Status {n}"
+
+            try:
+                client.send_status(my_status.encode(), timeout=10)
+                n += 1
+
+            except exceptions.ServerTookTooLong:
+                logging.error(f"Server timed out, context invalid: {e}")
```

Ruff

Undefined name `e`

(F821)